halyard-config/src/main/java/com/netflix/spinnaker/halyard/model/v1/providers/kubernetes/KubernetesAccount.java

```diff
@@ -32,6 +34,9 @@
   String context;
   String cluster;
   String user;
+  @ValidateField(validators = {ValidateFileExists.class})
+  String kubeconfigFile;
   List<String> namespaces = new ArrayList<>();
+  @ValidateField(validators = {ValidateNotNull.class})
```

Comment:
Looks to me like you're including an empty list anyway, so is there a case where this validator would ever fire? Perhaps a `ValidatorNotEmpty` is needed also/instead?